lib/logout.js

```diff
@@ -185,7 +187,8 @@ module.exports.logout = function (options) {
                 id: requestData.id,
                 serviceProviderLogoutURL: (session|| {}).serviceProviderLogoutURL || options.destination
               },
-              relayState: req.query.RelayState || (req.body && req.body.RelayState)
+              relayState: req.query.RelayState || (req.body && req.body.RelayState),
+              binding: session? session.binding : undefined,
```

Comment:
Also I'd suggest only adding `binding` to this state that we save if `session.binding` is set. Currently this will add `binding: undefined` always; given the `store` can be implemented by the user, they may not be expecting or able to handle this new property.